cli/cmd/check.go

```diff
@@ -225,8 +227,15 @@ func runChecksTable(wout io.Writer, hc *healthcheck.HealthChecker) bool {
 		}
 
 		fmt.Fprintf(wout, "%s %s\n", status, result.Description)
-		if result.Err != nil {
-			fmt.Fprintf(wout, "    %s\n", result.Err)
+		if err := result.Err; err != nil {
+			errMsg := fmt.Sprintf("%s", err)
+
+			// indent multi-line error message
```

Comment:
per other comments around multi-line error messages, i think this can be backed out.